src/kirby/icons/svg/QR.svg

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="24px" height="24px" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 55.1 (78136) - https://sketchapp.com -->
+    <title>QR</title>
```

Comment:
there is a slide problem with title and description, when using the icon om web. The title for the svg is show when mouse over is active, this override the mouse over for any parent element that are set. We have solved this with remove the title and decs for the svg file